README.md

```diff
@@ -71,6 +71,20 @@ yarn sqs-push [tiles.txt] [https://your-queue-url]
 
  The first argument, `tiles.txt`, is a line-delimited file containing your tile indices in the format `x-y-z` and the second argument is the URL of your SQS Queue. If you have a lot of tiles to push to the queue, it's best to run this script in the background or on a separate computer.  The maximum number of simultaneous inflight SQS requests can be set with the `PROMISE_THRESHOLD` environment variable.
 
+## Post processing
+
+Once the processing is complete, you can pull down the stored results as a simple CSV file.
+
+```sh
+DATABASE_URL='postgres://myusername:mypassword@your-db-string.rds.amazonaws.com:5432/ResultsDB' yarn download my_csv_filename.csv
```

Comment:
Can we construct the DB_URL for the user (should be either saved in env or available from config somehow

Reply:
It's not unwieldy now, but that would be a little nicer. Then we should also probably update the `yarn setup [DB_STRING]` step too. I'm no good with JS though, so might need help to do that

Reply:
ah yes, I guess that requires waiting for the CF output. Good to merge now then